App/Containers/Styles/RenameGroup.js

```diff
@@ -0,0 +1,43 @@
+import { StyleSheet } from 'react-native'
+
+import Colors from '../../Themes/Colors'
+import { color } from '../../styles'
+
+export default StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: Colors.modalColor,
```

Comment:
This is some old color module i'm trying to phase out. Use the new one like below.